tests/serialization/test_dag_serialization.py

```diff
@@ -648,6 +683,23 @@ def test_dag_serialized_fields_with_schema(self):
         dag_params: set = set(dag_schema.keys()) - ignored_keys
         self.assertEqual(set(DAG.get_serialized_fields()), dag_params)
 
+    def test_operator_subclass_changing_base_defaults(self):
```

Comment:
@kaxil this is the case that means we need to check the MRO for defaults.

I think storing all non-defaults is nicer anyway - we can show these in the UI that way